upload_analysis_files.py

```diff
@@ -73,6 +78,22 @@
         drive_client_wrapper.update_or_create(individuals_csv_input_path, individuals_csv_drive_dir,
                                               target_file_name=individuals_csv_drive_file_name,
                                               target_folder_is_shared_with_me=True, recursive=True)
+
+        log.info("Uploading Automated Analysis CSVs to Drive...")
+        paths_to_upload = glob(f"{automated_analysis_input_dir}/*.csv")
+        for i, path in enumerate(paths_to_upload):
+            log.info(f"Uploading CSV {i + 1}/{len(paths_to_upload)}: {path}...")
+            drive_client_wrapper.update_or_create(
```

Comment:
No need to change this immediately either as it's unrelated to this PR, but batch uploading a directory has a noticeable impact on pipeline performance, especially when there are many files to upload.

Reply:
great thanks waiting for pipeline infrastructure v.0.09. will update it